examples/basic_query/example_client.py

```diff
@@ -45,4 +45,4 @@ def signal_handler(signal, frame):
             predict(clipper_conn.get_query_addr(), np.random.random(200))
             time.sleep(0.2)
     except Exception as e:
-        clipper_conn.stop_all()
+        clipper_conn.stop_all()
```

Comment:
Revert this file

Reply:
It seems that I could not find the difference. Both seems to call clipper_conn.stop_all()